src/test/java/net/sf/jsqlparser/statement/alter/RenameTableStatementTest.java

```diff
@@ -88,8 +88,22 @@ public void testTableNamesFinder() throws JSQLParserException {
   @Test
   public void testValidator() throws JSQLParserException {
     String sqlStr = "RENAME oldTableName TO newTableName";
+    TestUtils.assertSqlCanBeParsedAndDeparsed(sqlStr, true);
+
+      // this needs to succeed
+      ValidationTestAsserts.validateNoErrors(sqlStr, 1, DatabaseType.ORACLE);
+
+      // this should fail when not supported in Postgres
+      ValidationTestAsserts.validateNoErrors(sqlStr, 1, DatabaseType.POSTGRESQL);
+
+      sqlStr = "ALTER TABLE public.oldTableName RENAME TO newTableName";
+      TestUtils.assertSqlCanBeParsedAndDeparsed(sqlStr, true);
+
+      //  should fail when not supported in Oracle
```

Comment:
Both syntaxes work in Oracle, and specifically, when trying to rename a table on a different schema, only the ALTER syntax works.

Reply:
Thanks, I have reworked the comments although the works remains on the Validator.